### DISCUSSION TITLE
whers dashboard

### DISCUSSION THREAD

**Discussion by ITAssassin8** [community/community#107461] [New to GitHub]
when you open the app you see yourself in a stupid surface homepage that you cant upload or change or communicate with anything or anyone 
while this app have insane abilities and tools that it seems to be hidden somewhere

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 ,

If you're looking to change anything on your GitHub profile, I recommend [this doc](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile). 

This site is certainly robust with its capabilities and features, our Skills courses break them down one by one and are a great place to start: https://skills.github.com/.
